### PR TITLE
chore(arrow): Support WKT and WKB geoarrow encodings (v4.1)

### PR DESCRIPTION
**Pull request by ibgreen**
@lixun910 Any chance you could generate and add some geoarrow files with WKB and WKT encodings?

### CODE REVIEW

**Comment by lixun910** [comment]
@ibgreen Yes, with the ogr2ogr scripts, I think I can add some geoarrow files with WKB and WKT encoding quickly. However, for big dataset, I do think it may be not efficient to use wkb/wkt in geoarrow format, since there will be extra parsing work involved.

```
ogr2ogr point_wkb.arrow point.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKB
ogr2ogr line_wkb.arrow line.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKB
ogr2ogr polygon_wkb.arrow polygon.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKB
ogr2ogr multipolygon_wkb.arrow multipolygon.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKB
ogr2ogr multipolygon_hole_wkb.arrow multipolygon_hole.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKB


ogr2ogr point_wkt.arrow point.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKT
ogr2ogr line_wkt.arrow line.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKT
ogr2ogr polygon_wkt.arrow polygon.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKT
ogr2ogr multipolygon_wkt.arrow multipolygon.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKT
ogr2ogr multipolygon_hole_wkt.arrow multipolygon_hole.arrow -f Arrow -lco COMPRESSION=NONE -lco GEOMETRY_ENCODING=WKT
```

**Comment by ibgreen** [comment]
@lixun910 thanks. During tests, I noticed that the generated files had encoding "WKB" and "WKT" rather then "geoarrow.wkb" and "geoarrow.wkt". but it was not hard to add some normalization code for that. 

I was able to get the tests running for WKT but not for WKB. I am not sure if the error is in how the binary WKB data is extracted from the vector cell, or if the format is somehow incompatible with loaders.gl WKB parser. 

Maybe you can take a quick look, just uncomment the WKB test cases...?

File: modules/arrow/src/geoarrow/convert-geoarrow-to-geojson-geometry.ts [hunk 1]
```diff
@@ -1,81 +1,95 @@
...
-    type: 'Feature',
-    geometry,
-    properties: {}
-  };
+
+  return geometry;
+}
+
+function arrowWKBToFeature(arrowCellValue: any) {
+  // The actual WKB array buffer starts from byteOffset and ends at byteOffset + byteLength
```
Comment (lixun910) [inline modules/arrow/src/geoarrow/convert-geoarrow-to-geojson-geometry.ts:74]: @ibgreen

Comment (ibgreen) [reply modules/arrow/src/geoarrow/convert-geoarrow-to-geojson-geometry.ts:74]: Duh, facepalm, of course...

**Pull request merged**